### ISSUE TITLE
A feature to see activities independently 

### ISSUE THREAD

**Issue by Ichiro786** [Emplexx/Aluminium#51]
![Untitled277_20241221190610](https://github.com/user-attachments/assets/8f78ec50-d8ef-4647-b359-7d23b2ea0a2e)
An option to view the activities of the people who u follow it can show the activities independently like organize it if u have used dantotsu u know what I'm talking about.. Something like this. 
![Screenshot_2024-12-22-08-42-50-61_8090ff421eb8a7caaa4bac331120a211](https://github.com/user-attachments/assets/d45573e0-79ef-4e04-9d0a-4ba97ae96111)

**Comment by Emplexx** [comment]
What is the purpose of this feature? Like do you want a quick way to see all activities from a single person you follow?

**Comment by Ichiro786** [comment]
> What is the purpose of this feature? Like do you want a quick way to see all activities from a single person you follow?

Yep.. Thats the main thing in this.. The traditional way of seeing isn't organized and too much cluttered i mean.. It has all the people's activities in same place

**Comment by Emplexx** [comment]
Okay, I'll need to think of a design for this though

**Comment by Ichiro786** [comment]
> Okay, I'll need to think of a design for this though 

Yeah sure it's ur choice i was just giving the idea of it.. U can change somethings in the ui.

**Comment by FujimiyaAmane-kun** [comment]
@Ichiro786 Leaving and having this through will get aluminum app way much more reliable and useful later... NICE IDEA by the way.
